lib/rules/jsx-no-bind.js

```diff
@@ -63,7 +62,7 @@ module.exports = {
     }],
   },
 
-  create: Components.detect((context) => {
+  create(context) {
```

Comment:
this doesn't seem right - we don't want this rule running on jsx that's outside of a detected component.